scripts/connect_clouddb_proxy.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+####################################
+# connect_clouddb_proxy.py
+#
+# This is a simple wrapper that looks up
+# the connection information in the config
+# and establishes a local proxy.
+#
+####################################
+
+
+import os
+import sys
+import subprocess
+from pprint import pprint
+
+from pocs.utils.config import load_config
+
+
+def main(instance_id, local_port, verbose=False):
+    proxy_cmd = os.path.join(os.environ['POCS'], 'bin', 'cloud_sql_proxy')
+    connection_str = '-instances={}=tcp:{}'.format(instance_id, local_port)
```

Comment:
`connection_str` seems like the wrong name for this var. Not sure if the following makes sense:
```
connection_str = '{}=tcp:{}'.format(instance_id, local_port)
instances_arg = '-instances={}'.format(connection_str)
```